main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TranslationsValidator.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import javax.inject.Inject;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsFeedContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsTableContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsTranslation;
+import org.mobilitydata.gtfsvalidator.table.GtfsTranslationTableContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsTranslationTableLoader;
+
+/**
+ * Validates that translations are provided in accordance with GTFS Specification. Supports two
+ * types of specifications * unofficial Google extension * official GTFS specification Checks that
```

Comment:
>unofficial Google extension

Is this extension documented somewhere that you could link to from here?

Also, it looks like the auto-formatting destroyed the bulleted list here - I'd suggest re-formatting in a way that keeps the list and complies with the Google Java Format.

Reply:
@lionel-nj As you work on profiles, perhaps pulling checks for the unofficial Google extension into a profile would be a good use case?

Reply:
Sorry for bad formatting!

Google translation.txt extension is hard to support in open-source since the "Google" translations.txt has different columns:

* `trans_id`
* `lang` (not `language`!)
* `translation`

Fields `table_name`, `field_name` and `language` are required in GTFS standard but not used in Google extension. So, it is not enough to add a plugin for validation of Google translations - instead, the schema has to be patched and officially required fields become optional in order to support Google translations.

Reply:
Yes, that was a bad copypaste from the internal code. Fixed.